### DISCUSSION TITLE
Low Text to HTML ratio

### DISCUSSION THREAD

**Discussion by anupkumarmaharjan** [sveltejs/kit#11475] [General]
### I have the project based on the svelte with 
Svelte: 4.2.0
@sveltejs/kit: ^1.24.0

and when it is run to calculate text to html ratio, the pages fail to threshold value which is 25% to 75%.

This is because the json is added on the HTML itself.

How can I achieve the high text to html ratio? 
The test is done with SEMrush.com and  https://www.siteguru.co/free-seo-tools/text-to-html-ratio

For eg. https://www.siteguru.co/free-seo-tools/text-to-html-ratio?url=https%3A%2F%2Fkit.svelte.dev%2F

<p><img width="580" alt="image" src="https://github.com/sveltejs/kit/assets/6189214/b6d2cb3a-8f47-4f80-83c2-7d83db58162f"></p>

### Can anyone help to adjust the Inline JSON Usage that Svelte uses to load the content faster?

**Comment by mtness** [comment]
To achieve a significantly better text-2-code ratio, maybe consider to just user plain, standard, dependancy-free html+css instead and use js only to enhance functionality ?!
